src/connectors/README.md

```diff
@@ -156,3 +156,61 @@ MyComponent.propTypes = {
 
 export default withUser(MyComponent);
 ```
+
+### withProductContext
+Connects provided component with a product context
+
+#### Props provided
+- `options (object)` - Current selected product options
+- `productId (string)` - Id of the current shown product
+- `variantId (string)` -  Id of the current selected variant
+- `conditioner (Function)` - Helper class for ProductCharacteristic component
+
+
+#### Example usage
+```jsx
+import React from 'react';
+import { withProductContext } from '@shopgate/pwa-extension-kit/connectors';
+
+const MyComponent = ({ options, productId, variantId, conditioner }) => {
+  
+  return (
+    <ProductCharacteristics
+        productId={productId}
+        variantId={variantId}
+        render={this.renderer}
+        conditioner={conditioner}
+        finishTimeout={200}
+      />
+  );
+}
+
+export default withProductContext(MyComponent);
+```
+
+### withThemeComponents
+Connects provided component with some components provided by the Theme.
```

Comment:
Some background info would be helpful here. "with components provided by the Theme. Theme components are theme specific - maintain functionality with theme specific UI"

Reply:
done